ironfish/src/defaultNetworkDefinitions.ts

```diff
@@ -61,6 +67,7 @@ export const DEVNET = `
         "targetBlockTimeInSeconds": 60,
         "targetBucketTimeInSeconds": 10,
         "maxBlockSizeBytes": 524288,
-        "minFee": 0
+        "minFee": 0,
+        "enableAssetOwnership": 9999999
```

Comment:
I'd say that for DEVNET we can set it to 0 from the start. Or we can update this later, up to you

Reply:
Yeah - that's the plan. We can't set it that low yet because the code doesn't exist to make the tests pass, and we'll have to regenerate all the test fixtures so it'll be a PR to follow soon. Here's the commit from the queue: https://github.com/iron-fish/ironfish/commit/8d763c075c91d143e38c3b7de105e478606d9a3a